app/src/main/java/com/p2p/presentation/home/games/Game.kt

```diff
@@ -4,6 +4,7 @@ import androidx.annotation.DrawableRes
 import androidx.annotation.StringRes
 import com.p2p.R
 
-enum class Game(@StringRes val nameRes: Int, @DrawableRes val imageRes: Int) {
-    TUTTI_FRUTTI(R.string.games_tutti_frutti, R.drawable.tutti_frutti),
+enum class Game(@StringRes val nameRes: Int, @DrawableRes val iconRes: Int) {
+
+    TUTTI_FRUTTI(R.string.games_tutti_frutti, R.drawable.ic_tutti_frutti),;
```

Comment:
Ese punto y coma no va :)
```suggestion
    TUTTI_FRUTTI(R.string.games_tutti_frutti, R.drawable.ic_tutti_frutti),
```